class NotFoundError(Exception):
    pass

class NotURLError(Exception):
    pass